tensorflow_lite_support/ios/test/task/vision/image_classifier/TFLImageClassifierTests.m

```diff
@@ -0,0 +1,135 @@
+
+  TFLImageClassifier *imageClassifier =
+      [TFLImageClassifier imageClassifierWithOptions:imageClassifierOptions error:nil];
+  XCTAssertNotNil(imageClassifier);
+  GMLImage *gmlImage = [self imageFromBundleWithName:@"burger_crop" ofType:@"jpg"];
+
+  TFLClassificationResult *classificationResults = [imageClassifier classifyWithGMLImage:gmlImage
+                                                                                   error:nil];
+  XCTAssertTrue([classificationResults.classifications count] > 0);
+  XCTAssertTrue([classificationResults.classifications[0].categories count] > 0);
+
+  TFLCategory *category = classificationResults.classifications[0].categories[0];
+  XCTAssertTrue([category.label isEqual:@"cheeseburger"]);
+}
+
+- (void)testModelOptionsWithMaxResults {
+  TFLImageClassifierOptions *imageClassifierOptions =
+      [[TFLImageClassifierOptions alloc] initWithModelPath:self.modelPath];
+  int maxResults = 3;
+  imageClassifierOptions.classificationOptions.maxResults = maxResults;
+
+  TFLImageClassifier *imageClassifier =
+      [TFLImageClassifier imageClassifierWithOptions:imageClassifierOptions error:nil];
+  XCTAssertNotNil(imageClassifier);
+
+  GMLImage *gmlImage = [self imageFromBundleWithName:@"burger_crop" ofType:@"jpg"];
+
+  TFLClassificationResult *classificationResults = [imageClassifier classifyWithGMLImage:gmlImage
+                                                                                   error:nil];
+  XCTAssertTrue([classificationResults.classifications count] > 0);
+  XCTAssertTrue([classificationResults.classifications[0].categories count] > 0);
+
+  TFLCategory *category = classificationResults.classifications[0].categories[0];
+  XCTAssertTrue([category.label isEqual:@"cheeseburger"]);
+}
+
+- (void)testInferenceWithBoundingBox {
+  TFLImageClassifierOptions *imageClassifierOptions =
+      [[TFLImageClassifierOptions alloc] initWithModelPath:self.modelPath];
+  int maxResults = 3;
+  imageClassifierOptions.classificationOptions.maxResults = maxResults;
+
+  TFLImageClassifier *imageClassifier =
+      [TFLImageClassifier imageClassifierWithOptions:imageClassifierOptions error:nil];
+  XCTAssertNotNil(imageClassifier);
+
+  GMLImage *gmlImage = [self imageFromBundleWithName:@"burger_crop" ofType:@"jpg"];
+
+  CGRect roi = CGRectMake(20, 20, 200, 200);
+  TFLClassificationResult *classificationResults = [imageClassifier classifyWithGMLImage:gmlImage
+                                                                        regionOfInterest:roi
+                                                                                   error:nil];
+  XCTAssertTrue([classificationResults.classifications count] > 0);
+  XCTAssertTrue([classificationResults.classifications[0].categories count] > 0);
+
+  TFLCategory *category = classificationResults.classifications[0].categories[0];
+  XCTAssertTrue([category.label isEqual:@"cheeseburger"]);
```

Comment:
Same here and the test below.  Test `score` please.

Reply:
Fixed